From 3b35b4c283d4dc95cf92d4a5e15f1a048eb4013f Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Mon, 4 Apr 2016 18:41:55 +0300
Subject: [PATCH 12/15] configs: r8a7795_salvator-x: Enable RPC HyperFlash
 support

This enables flash commands along with the RPC HyperFlash support.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/r8a7795_salvator-x.h | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/include/configs/r8a7795_salvator-x.h b/include/configs/r8a7795_salvator-x.h
index d5c4c93..db5eae5 100644
--- a/include/configs/r8a7795_salvator-x.h
+++ b/include/configs/r8a7795_salvator-x.h
@@ -28,10 +28,13 @@
 
 /* [A] Hyper Flash */
 /* use to RPC(SPI Multi I/O Bus Controller) */
+#define CONFIG_RPC_HYPERFLASH
 
-	/* underconstruction */
+#define CONFIG_SYS_FLASH_BASE		CONFIG_SYS_RPC_FLASH_BASE
+#define CONFIG_SYS_MAX_FLASH_BANKS	1
+#define CONFIG_SYS_MAX_FLASH_SECT	256
 
-#define CONFIG_SYS_NO_FLASH
+#define CONFIG_CMD_FLASH
 
 /* Ethernet RAVB */
 #define CONFIG_RAVB
-- 
1.9.3

